TEAM_STRUCTURES.md

```diff
@@ -75,6 +75,7 @@ A byproduct of both of these team structures achieves another important goal: ma
 - **Working Group Mailing List**: js-ipfs-wg@ipfs.io
 - **[Alan Shaw](https://github.com/alanshaw/): Captain**
 - **`To be confirmed`: TPM**
+- WG Repos Badge - [![](https://img.shields.io/badge/Working%20Group-JS%20CORE-%23ffd600.svg?style=flat-square)](https://github.com/ipfs/js-ipfs)
```

Comment:
Note to self: Before merging, host these badges in ipfs/team-mgmt so that we can change in one location and update in all repos.

Reply:
Official JS colour?:

```suggestion
- WG Repos Badge - [![](https://img.shields.io/badge/Working%20Group-JS%20CORE-%23f7df1e.svg?style=flat-square)](https://github.com/ipfs/js-ipfs)
```